src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/EnergyNetComponent.java

```diff
@@ -199,4 +200,17 @@ default void removeCharge(@Nonnull Location l, int charge) {
         }
     }
 
+    /**
+     * Override this method to display additional information to the {@link Player} when a
+     * {@link io.github.thebusybiscuit.slimefun4.implementation.items.electric.gadgets.Multimeter} is used.
```

Comment:
Why is this written out...?
Add an import instead, you use the short version below anyway.
```suggestion
     * {@link Multimeter} is used.
```